postgres/postgresql-entrypoint.sh

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env bash
```

Comment:
Would it be helpful at all to put a note at the top here about where this script came from / what it was based on?

Reply:
good call, will do that now